### PR TITLE
GDB-8810 introduces iri autocomplete select and integrates it in the acl view

### PR DESCRIPTION
**Pull request by svilenvelikov**
## What
IRI autocomplete component is implemented and integrated it in the ACL management view.

## Why
This improves the user experience allowing easier completion of triples when editing acl rules.

## How
Implemented the autocomplete component based on the rdf-resource-search component.
Integrated the component in the ACL management view.

### CODE REVIEW

File: src/js/angular/aclmanagement/controllers.js [hunk 1]
```diff
@@ -12,9 +13,13 @@ angular
...
     .controller('AclManagementCtrl', AclManagementCtrl);
 
-AclManagementCtrl.$inject = ['$scope', '$location', 'toastr', 'AclManagementRestService', '$repositories', '$translate', 'ModalService'];
+AclManagementCtrl.$inject = ['$scope', '$location', 'toastr', 'AclManagementRestService', '$repositories', '$translate', 'ModalService', 'RDF4JRepositoriesRestService', 'AutocompleteRestService'];
 
-function AclManagementCtrl($scope, $location, toastr, AclManagementRestService, $repositories, $translate, ModalService) {
+function AclManagementCtrl($scope, $location, toastr, AclManagementRestService, $repositories, $translate, ModalService, RDF4JRepositoriesRestService, AutocompleteRestService) {
+
+    $scope.contextValue = undefined;
+    $scope.subjectValue = 'http://www.w3.org/TR/2003/PR-owl-guide-20031209/wine#Wine';
```
Comment (yordanalexandrov) [inline src/js/angular/aclmanagement/controllers.js:21]: Should those be hardcoded?

**Review by yordanalexandrov — APPROVED** [review]

File: src/js/angular/aclmanagement/controllers.js (hunk 1 above)
Comment (svilenvelikov) [reply src/js/angular/aclmanagement/controllers.js:21]: No. These are leftovers. Will remove them.

**Review by yordanalexandrov — APPROVED** [review]

**Review by boyan-tonchev — APPROVED** [review]

**Comment by svilenvelikov** [comment]
Jenkins retry build

**Comment by svilenvelikov** [comment]
Jenkins retry build

**Comment by svilenvelikov** [comment]
Jenkins retry build

**Review by boyan-tonchev — APPROVED** [review]

**Pull request merged**